browser/component_updater/brave_component_contents_verifier.cc

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/component_updater/brave_component_contents_verifier.h"
+
+#include <algorithm>
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "base/files/file_path.h"
+#include "base/files/file_util.h"
+#include "base/functional/bind.h"
+#include "brave/components/brave_component_updater/browser/component_contents_verifier.h"
+#include "extensions/buildflags/buildflags.h"
+
+#if BUILDFLAG(ENABLE_EXTENSIONS)
+
+#include "base/command_line.h"
```

Comment:
These headers are not all guarded by `enable_extension` in gn. Don't match headers with guards based on how they are used in the file, they should always match the guards in gn.